libs/ui/src/components/NeoLoading/NeoLoading.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <o-loading
+    :active.sync="isLoadingActive"
+    :can-cancel="canCancel"
+    :overlay-class="'neo-loading-overlay'"
```

Comment:
```suggestion
    overlay-class="neo-loading-overlay"
```

or maybe create `has-overlay` prop with default `true`